src/event/cloudevent.ts

```diff
@@ -195,4 +195,12 @@ See: https://github.com/cloudevents/spec/blob/v1.0/spec.md#type-system`);
   ): CloudEvent {
     return new CloudEvent(Object.assign({}, this.toJSON(), options) as CloudEvent, strict);
   }
+
+  /**
+   * The native `console.log` value of the CloudEvent.
+   * @return The string representation of the CloudEvent.
+   */
+  [Symbol.for("nodejs.util.inspect.custom")]() {
```

Comment:
This is a pretty clever approach. :)

Reply:
Thanks! Came from: https://nodejs.org/api/util.html#util_util_inspect_custom